bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/servlets/FormMetaDataDataSourceServlet.java

```diff
@@ -193,6 +195,16 @@ private List<Resource> getDataSourceResources(HttpServletRequest request, Resour
                     metaDataList = formMetaData.getPrefillActions();
                     resources = this.getResourceListFromComponentDescription(metaDataList, resourceResolver);
                     break;
+                case CUSTOM_PROPERTIES:
+                    policy = ComponentUtils.getPolicy((String) request.getAttribute(Value.CONTENTPATH_ATTRIBUTE),
```

Comment:
minor: you may initialise the policy at once in line 157 above switch case since it will be common.